### PR TITLE
Update config.php

### PR DESCRIPTION
**Pull request by einar-lanfranco**
hot FIX! missing space in proc_open cause timeouts

### CODE REVIEW

File: intelmq-manager/php/config.php [hunk 1]
```diff
@@ -29,7 +29,7 @@
         'positions' => '/opt/intelmq/etc/manager/positions.conf',
     );
     # get paths from intelmqctl directly if it works
-    $proc = proc_open($c . "--type json debug --get-paths", [
+    $proc = proc_open($c . " --type json debug --get-paths", [
```
Comment (wagner-certat) [inline intelmq-manager/php/config.php:32]: Thanks for reporting that bug!
However, I propose to instead use
```python
sprintf($CONTROLLER_JSON, "debug --get-paths")
```
as command to be in line with the other command-compositions in the controller file.

File: intelmq-manager/php/config.php [hunk 2]
```diff
@@ -39,13 +39,10 @@
...
-        $FILES['harmonization'] = $output['HARMONIZATION_CONF_FILE'];
-        $FILES['pipeline'] = $output['PIPELINE_CONF_FILE'];
-        $FILES['runtime'] = $output['RUNTIME_CONF_FILE'];
-        $FILES['system'] = $output['SYSTEM_CONF_FILE'];
-        $FILES['positions'] = $output['CONFIG_DIR'] . "/manager/positions.conf";
-    }
+	    $output = json_decode($paths_stdout);
+	    foreach($output->paths as $path){
+		    $FILES[$path[0]]=$path[1];
+	    }
```
Comment (wagner-certat) [inline intelmq-manager/php/config.php:45]: Why? This sets variables which are never used. The result is an array like this:
```php
Array
(
    [bots] => /opt/intelmq/etc/BOTS
    [defaults] => /opt/intelmq/etc/defaults.conf
    [harmonization] => /opt/intelmq/etc/harmonization.conf
    [pipeline] => /opt/intelmq/etc/pipeline.conf
    [runtime] => /opt/intelmq/etc/runtime.conf
    [system] => /opt/intelmq/etc/system.conf
    [positions] => /opt/intelmq/etc/manager/positions.conf
    [BOTS_FILE] => /opt/intelmq1/etc/BOTS
    [DEFAULTS_CONF_FILE] => /opt/intelmq1/etc/defaults.conf
    [HARMONIZATION_CONF_FILE] => /opt/intelmq1/etc/harmonization.conf
    [PIPELINE_CONF_FILE] => /opt/intelmq1/etc/pipeline.conf
    [RUNTIME_CONF_FILE] => /opt/intelmq1/etc/runtime.conf
    [VAR_RUN_PATH] => /opt/intelmq1/var/run/
    [STATE_FILE_PATH] => /opt/intelmq1/var/lib/state.json
    [DEFAULT_LOGGING_PATH] => /opt/intelmq1/var/log/
    [__file__] => /home/sebastianw/dev/intelmq/intelmq/bin/intelmqctl.py
    [CONFIG_DIR] => /opt/intelmq1/etc/
    [ROOT_DIR] => /opt/intelmq1/
)
```

**Review by ghost — CHANGES_REQUESTED** [review]

File: intelmq-manager/php/config.php (hunk 2 above)
Comment (wagner-certat) [reply intelmq-manager/php/config.php:45]: I'm working on a patch for intelmqctl to provide the data as dictionary, not as list

Comment (einar-lanfranco) [reply intelmq-manager/php/config.php:45]: Yes, there are a couple of variable it never uses, but some of them are used in the others scripts, like php/load_configs.php.
Im sure we could do it better but i dont know all the app and as i noticed you are changing to python i just wrote a simple fix!

**Review by einar-lanfranco — COMMENTED** [review]
Sounds better

File: intelmq-manager/php/config.php (hunk 2 above)
Comment (wagner-certat) [reply intelmq-manager/php/config.php:45]: What I meant is, that your your wrote the string `/opt/intelmq1/etc/BOTS` (as an example for an alternative root) to the key `BOTS_FILE`, which is never used, but should be `bots` instead. Same for all other variables.

**Comment by wagner-certat** [comment]
I want to do a 2.2.1 release for IntelMQ and Manager this week to get this fix out.

Thanks again for reporting and proposing a fix!

**Pull request merged**